client/src/client.rs

```diff
@@ -90,25 +90,28 @@ impl Client {
         }
     }
 
-    /// Builds transaction out of supplied instructions.
+    /// Builds transaction out of supplied instructions or wasm.
     ///
     /// # Errors
     /// Fails if signing transaction fails
     pub fn build_transaction(
         &self,
-        instructions: Vec<Instruction>,
+        instructions: Executable,
```

Comment:
```suggestion
        instructions: Execute,
```
Minor point, but -able traits are not very rust-like.

Reply:
The thing is this is not a trait. It is an enum. So `Executable` seems good to me.